jetson/face_detector_threaded.py

```diff
@@ -293,17 +296,18 @@ def encryptFrame(img, boxes):
     g.eval()
     class_names = ['Glasses', 'Goggles', 'Neither']
 
-    cap = VideoCapturer() #Instantiate Video Capturer object
-    detector = FaceDetector(trained_model=args.trained_model, cuda=args.cuda and torch.cuda.is_available(), set_default_dev=True) #Instantiate Face Detector object
-    cl = Classifier(g) #Instantiate Classifier object
+    cap = VideoCapturer()  # Instantiate Video Capturer object
+    detector = FaceDetector(trained_model=args.trained_model, cuda=args.cuda and torch.cuda.is_available(),
+                            set_default_dev=True)  # Instantiate Face Detector object
+    cl = Classifier(g)  # Instantiate Classifier object
 
     while True:
         start_time = time.time()
 
         frame = cap.get_frame()
         boxes = detector.detect(frame)
 
-        encryptedImg = frame.copy() #copy for creating encrypted image
+        encryptedImg = frame.copy()  # copy for creating encrypted image
```

Comment:
tell me why you need a copy not that you are making a copy

Reply:
@jbagnara is `encryptedImg` the one that gets encrypted and stored in the database while `frame` is what's displayed on the screen?